packages/phone-number-privacy/combiner/src/bls/bls-cryptography-client.ts

```diff
@@ -63,22 +65,21 @@ export class BLSCryptographyClient {
       logger.error(error)
       // Verify each signature and remove invalid ones
       // This logging will help us troubleshoot which signers are having issues
-      const verifySigReqs = this.unverifiedSignatures.map((unverifiedSignature) => {
-        return this.verifySignature(blindedMessage, unverifiedSignature, logger!)
+      this.unverifiedSignatures.map((unverifiedSignature) => {
```

Comment:
Nit: This should be .forEach since we no longer need to write the promises to the unverifiedSignatures array